packages/shared-core/src/filters.ts

```diff
@@ -351,7 +427,7 @@ export const luceneSort = (docs, sort, sortOrder, sortType = "string") => {
  * @param docs the data
  * @param limit the number of docs to limit to
  */
-export const luceneLimit = (docs, limit) => {
+export const luceneLimit = (docs: string | any[], limit: string) => {
```

Comment:
This seems a bit strange. How could docs by a string or an array of objects?

Should limit not be a number?